changelog

```diff
@@ -1,3 +1,400 @@
+      * The descriptions of the Heliarch reactors now indicate that they used anti-matter catalyzed fusion technology. (@Arachi-Lover)
+      * Governments that reuse colors now refer to named "stock" colors instead of providing their own color literals. (@warp-core)
+      * Added a reference to the player's manual to the credits. (@hmglasgow)
+      * Separated planet and system definitions into their own files. (@Amazinite)
+      * Brighten some government colors to improve map readability. (@TomGoodIdea)
+      * Used custom penalties to improve Remnant and Quarg reactions to actions against some foreign governments. (@Hurleveur)
+      * Added departure distances to systems with Quarg Ringworlds. (@RisingLeaf)
+      * Void Sprites now drop Void Sprite Parts. (@Quantumshark, @Saugia)
+  * Game mechanics:
+    * New mechanics:
+      * New autoconditions:
+        * Pilot name. (@RisingLeaf)
+        * Ships, total ships, ship model, outfits, visited. (@Amazinite)
+        * Flagship planet attribute. (@rovermicrover)
+        * Several date related conditions. (@warp-core)
+        * Conditions for the number of hyperspace jumps to another planet or system. (@warp-core)
+        * Flagship attribute. (@warp-core)
+        * Flagship landed. (@Hurleveur)
+        * Entered system by and previous system. (@Hurleveur, @Amazinite)
+      * Illegal outfits and cargo:
+        * "scan brightness" attribute, increases the chance an outfit is found on a scan. (@quyykk)
+        * "scan concealment" attribute, reduces the chance an outfit is found on a scan. (@quyykk)
+        * Illegal and attrocity goods can now be defined individually for different governments. (@Hurleveur, @tibetiroka)
+      * Added the option for a hyper, scram, or jump drive to have variable fuel cost based on the mass of the ship using it. (@Ferociousfeind)
+      * Added a new "merciful" personality that causes ships to avoid attacking hostile ships that are fleeing. (@Hurleveur, @quyykk)
+      * Added "drag reduction" and "inertia reduction" attributes. (@Azure3141)
+      * Systems can now define a "departure distance", a minimum distance a ship must be from the centre to jump to another system. (@RisingLeaf, @petervdmeer, @quyykk)
+      * When taking off from a planet with an outfitter with more outfits in cargo than there is space for, store them instead of selling them. (@warp-core)
+      * Ship variant definitions can now remove all the bays from the base definition. (@daggs1)
+      * It is now possible to offer a mission in the shipyard or outfitter. (@rovermmicrover)
+      * A system definition can now include a custom value for the invisible fence radius. (@a-random-lemurian)
+      * TODO: find a concise way to describe the foreign penalty thing for governments. (@Hurleveur)
+      * Missions can now define a "to accept" condition set. (@rovermicrover)
+      * Governments can use named "stock" colors instead of always defining a color literal. (@warp-core)
```

Comment:
Also on line 257. Is having both intentional?

If so, probably should have the exact same wording in both. It will be repetitive, yes, but should also help highlight it's the same change instead of possibly making people think it's two different but related/similar changes.